apps/cookbook/src/app/examples/card-example/examples/banner.ts

```diff
@@ -0,0 +1,45 @@
+import { Component } from '@angular/core';
+import { noop } from 'rxjs';
+
+const config = {
+  selector: 'cookbook-card-example-banner',
+  codeSnippet: `<kirby-card  hasPadding="true" (click)="noop()">
+  <kirby-card-footer [hasPadding]="false">
+  <kirby-item disclosure="arrow-more">
+    Banner call to action
+  </kirby-item>
+</kirby-card-footer>
+
+</kirby-card>`,
+  template: `<kirby-card hasPadding="true" (click)="noop()">
+  <kirby-card-footer [hasPadding]="false">
+    <kirby-item disclosure="arrow-more">
+      Banner call to action
+    </kirby-item>
+  </kirby-card-footer>
+</kirby-card>`,
+  style: `@use '@kirbydesign/designsystem/src/scss/utils';
```

Comment:
We move around our files a little bit when distributing the kirby package, so consumers of the code will actually be able to find the styling under this path:

```suggestion
  style: `@use '@kirbydesign/designsystem/scss/utils';
```